### DISCUSSION TITLE
Delete DB from google nightscout instance

### DISCUSSION THREAD

**Discussion by jgslade** [NightscoutFoundation/xDrip#2899] [Q&A]
I accidentally imported the data from my old nightscout into the new one I setup on google, so all my data is doubled. Is there a way to remove the data or de-duplicate it without having to destroy the whole thing and start over from scratch?

**Comment by Navid200** [comment]
Which method did you use for the import?

**Reply by Navid200** [reply]
I don't believe there is any duplication.  You will see expected errors identifying items that are already present.  But, they are not duplicated.

If you tell us how you imported, we can try to recreate the problem.
So, please explain why you think you have duplication.

**Reply by jgslade** [reply]
I used the import option from the menu, then provided the url for my old NS site. When doing a day to day report on both sites the graph looks the same, but the numbers are twice as high on the new site as the old site.

I did find some documentation on mongodb and used the db.collection.deleteMany({}) command to remove the entries for the non auth collections then reimported, which kinda worked, but I do have a bit of duplication for the time it took to reimport.

**Reply by Navid200** [reply]
> , but the numbers are twice as high ...

What do you mean?  Do you mean the number of points is correct?  But, the glucose readings are twice as they should be?!

**Comment by jgslade** [comment]
No the insulin and carbs are twice, ie the original said I used 55 units
for the day the new one said I used 110. But on the graph the basal lines
match up and the bolus bubbles are the same.

**Reply by Navid200** [reply]
I have asked our experts to help.  But, they are scratching their heads and have no suggestions I guess.

All I can suggest is to ask in a forum about mongodb.
We are using the standard mongodb.  It is not like we have our own version.  So, anything you experience is what everyone else experiences using mongodb.

**Comment by jgslade** [comment]
Ok, thanks for looking into it.
